app/src/routes/v1/email.js

```diff
@@ -22,6 +23,13 @@ emailRouter.post('/', [
     }).send(res);
   }
 
+  // Fail quietly (no exception raised). If this fails let the user request go through normally (log out failure)
+  try {
+    await github.createRequestIssue(req.body.applicationAcronym, req.body.comments, req.body.from, req.body.idir);
```

Comment:
I don't think we even need to await on this as we're not using this result anywhere else. This can be a fully asynchronous call.